server.js

```diff
@@ -2648,6 +2648,51 @@ cache(function(data, match, sendBadge, request) {
   });
 }));
 
+// GitHub all releases download count integration.
+camp.route(/^\/github\/dt\/([^\/]+)\/([^\/]+)\.(svg|png|gif|jpg|json)$/,
```

Comment:
Could you merge this with `github/downloads` please? We can consider that a tag named `total` actually means that we should look at all tags.